### ISSUE TITLE
Cannot access anaconda-mode (0.1.9) python package

### ISSUE THREAD

**Issue by yfy-** [proofit404/anaconda-mode#349]
I am trying to install anaconda-mode on a computer that is not directly connected to the internet. I tried to manually install anaconda-mode python package but links; [https://pypi.python.org/simple/anaconda_mode](url) , [https://pypi.python.org/simple/anaconda-mode](url), [https://pypi.org/project/anaconda_mode](url) all seem to be broken. My ~/.emacs.d/anaconda-mode/ is empty. How can I access the package?

Thanks

**Comment by beyondpie** [comment]
hi, you want to install anaconda-mode for emacs, right?

**Comment by yfy-** [comment]
Yes, I need it for emacs.

**Comment by beyondpie** [comment]
[https://melpa.org/#/anaconda-mode](how about this link on Melpa)

**Comment by proofit404** [comment]
Python package is obsolete for two years. Its content was embedded completely into Emacs package.

**Comment by yfy-** [comment]
> how about this link on Melpa
> https://melpa.org/#/anaconda-mode

But this is just the .el file. As I said earlier, anaconda complains that I don't have the python package.



> Python package is obsolete for two years. Its content was embedded completely into Emacs package.

Hmm, strange. I installed anaconda-mode recently and it was searching for the python package in the links I've written in my initial comment. Nevertheless, I've found the package from an older emacs installation and it works as expected.

Thanks
